src/test/rpc/Transaction_test.cpp

```diff
@@ -792,6 +862,8 @@ class Transaction_test : public beast::unit_test::suite
         testCTIDRPC(features);
         test::jtx::forAllApiVersions(
             std::bind_front(&Transaction_test::testRequest, this, features));
+        test::jtx::forAllApiVersions(
+            std::bind_front(&Transaction_test::testBinaryRequest, this));
```

Comment:
Why not just:
```
test::jtx::forAllApiVersions(
           std::bind_front(&Transaction_test::testRequest, this, features)
           std::bind_front(&Transaction_test::testBinaryRequest, this));
```
instead of calling `test::jtx::forAllApiVersions` twice ?

Reply:
good point, will do